hadoop-plugin/src/main/groovy/com/linkedin/gradle/azkaban/AzkabanHelper.groovy

```diff
@@ -144,7 +143,6 @@ class AzkabanHelper {
    * @return The parsed response
    */
   static String parseResponse(String response) {
-    logger.lifecycle("AzkabanHelper: parseResponse");
     String newline = System.getProperty("line.separator");
     return HtmlUtil.toText(response).replaceAll("azkaban.failure.message=", newline + newline);
```

Comment:
When you run the ligradle azkabanUpload task, did you notice that the URL that gets displayed to the user that should link to the project has an extra slash in it after the port number, e.g."8443://manager"? This makes it so when you browse to that URL, it doesn't actually go to the Azkaban project page. Could you track down that string that gets displayed to the user and replaceAll("//manager") with "/manager"?